docs/news.rst

```diff
@@ -2,11 +2,55 @@
 Release Notes
 =============
 
+
+2.2.0 (2020-08-28)
+==================
+
+Highlights
+----------
+
+- Files that are part of
+  `syncFromSynapse <https://python-docs.synapse.org/build/html/synapseutils.html#synapseutils.sync.syncFromSynapse>`__
+  and
+  `syncToSynapse <https://python-docs.synapse.org/build/html/synapseutils.html#synapseutils.sync.syncToSynapse>`__
+  operations (:code:`synapse get -r` and :code:`synapse sync` in the command line client, respectively) are
+  transferred in in parallel threads rather than serially, substantially improving the performance of these operations.
+
```

Comment:
```suggestion
- Table metadata from `synapse get -q` is automatically downloaded to a users working directory instead of to the Synapse cache (a hidden folder).
- Users can now pass their API key to `synapse login` in place of a password. 
```